src/app/shared/services/sentry/sentry.service.ts

```diff
@@ -0,0 +1,64 @@
+import { APP_INITIALIZER, ErrorHandler, Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { catchError, map } from 'rxjs/operators';
+import { of } from 'rxjs';
+import * as Sentry from '@sentry/angular';
+import { Router } from '@angular/router';
+import { EnvService } from '../env/env.service';
+
+@Injectable()
+export class SentryService {
+  constructor(private http: HttpClient, private envService: EnvService) {}
+
+  public load() {
+    return this.http
+      .get<{ dsn: string }>('sentry-config.json', {
+        headers: { 'Cache-Control': 'no-store' },
+      })
```

Comment:
```suggestion
  /**
     * Storying DSN in github secrets to make it somewhat more difficult to obtain and so less likely that it will be abused.
     * https://github.com/getsentry/sentry-docs/pull/1723/files#r577851784
     * This is why the values is written to a file using the GHA workflow and is loaded here
     */
  public load() {
    return this.http
      .get<{ dsn: string }>('sentry-config.json', {
        headers: { 'Cache-Control': 'no-store' },
      })
```

Reply:
Added this comment. Thanks! 👍